benchmarks/benchmark-micronaut-data-jdbc/build.gradle

```diff
@@ -15,3 +15,9 @@ dependencies {
     runtimeOnly "com.h2database:h2"
     runtimeOnly mn.snakeyaml
 }
+
+graalvmNative {
+    metadataRepository {
+        uri("https://github.com/oracle/graalvm-reachability-metadata/releases/download/0.2.7/graalvm-reachability-metadata-0.2.7.zip")
```

Comment:
Do we need this?

Reply:
I think we can avoid testing benchmarks

Reply:
Also, that url will be completely removed once we have all metadata used by the latest Native Build Tool plugin